src/libambit/pmem20.c

```diff
@@ -93,7 +93,7 @@ int libambit_pmem20_log_init(libambit_pmem20_t *object, uint32_t mem_start, uint
 
     // Set memory structure
     object->log.mem_start = mem_start;
-    object->log.mem_size = mem_size;
+    object->log.mem_size = (mem_size) ?:-1; // mem_size is 0 ??
```

Comment:
The response is in the message description. We initialize log.mem_size with Exercise log size (got from memory map), and checks for size overflow, but it's 0 on my version which lead parser to fail.